monai/transforms/utility/array.py

```
@@ -1950,3 +1951,39 @@ def inverse(self, data: torch.Tensor) -> torch.Tensor:
+
+        flatten = FlattenSequence()
+        data = [[1, 2], [3, 4], [5, 6]]
+        print(flatten(data))
+        [1, 2, 3, 4, 5, 6]
+
+    """
+
+    def __init__(self):
+        super().__init__()
+
+def __call__(self, data: list | tuple | Any) -> list | tuple | Any:
```

Ruff (0.13.3)

Unused function argument: `self`

(ARG001)